src/main/java/net/dv8tion/jda/api/entities/Role.java

```diff
@@ -271,4 +285,33 @@ default RoleAction createCopy()
      */
     @Nonnull
     JDA getJDA();
+
+
+    /**
+     * Enum used to differentiate between the different types of Guild Roles.
+     */
+    enum Type
+    {
+        /**
+         * A basic, user-created role.
+         */
+        NONE,
```

Comment:
In case of this enum showing Types of Roles, NONE is not applicable. its not a NONE Role. NORMAL like in initial draft would fit better

Reply:
Back to NORMAL. I thought normal felt better too